README.md

```diff
@@ -59,7 +59,8 @@ Role Variables
       defined with the following dict:
         - `pool`: Name or UUID of the storage pool from which the volume should be
           allocated.
-        - `name`: Name to associate with the volume being created.
+        - `name`: Name to associate with the volume being created; include extension if you would like images created with one.
```

Comment:
For `file` type volumes, include the extension if you would like images created with one.

Reply:
I've imported these words in to 6356e58